Doc/library/urllib.request.rst

```diff
@@ -181,6 +181,12 @@ The :mod:`urllib.request` module defines the following functions:
       >>> url2pathname(url.removeprefix('file:'))
       'C:\\Program Files'
 
+   .. versionchanged:: next
```

Comment:
There is a "versionchanged" below so maybe put this one after "Windows drive letters are no longer converted to uppercase [...]"? (so that if this doesn't land in 3.15, we don't have a versionchanged 3.15 followed by a 3.14?)